Hearthstone/app/src/main/java/com/example/hearthstone/data/network/api/HearthstoneApi.kt

```diff
@@ -12,4 +14,18 @@ interface HearthstoneApi {
         @Header("X-RapidAPI-Key") apiKey: String = "",
         @Header("X-RapidAPI-Host") apiHost: String = "omgvamp-hearthstone-v1.p.rapidapi.com"
     ): Response<HearthstoneModel>
+
+    @GET("cards/classes/{name}")
+    suspend fun getCardsByClass(
+        @Header("X-RapidAPI-Key") apiKey: String = "",
+        @Header("X-RapidAPI-Host") apiHost: String = "omgvamp-hearthstone-v1.p.rapidapi.com",
+        @Path("name") className: String
+    ): Response<List<HSCardsByClassModel>>
+
+    @GET("cards/search/{name}")
+    suspend fun getCardsByName(
+        @Header("X-RapidAPI-Key") apiKey: String = "",
+        @Header("X-RapidAPI-Host") apiHost: String = "omgvamp-hearthstone-v1.p.rapidapi.com",
```

Comment:
you can create a variable like val apiHost: String ="omgvamp-hearthstone-v1.p.rapidapi.com", and use it instead of typing it every time you need it